src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +75,183 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+
+  it('should update of new mappings row data', () => {
```

Comment:
In the above description, when should the update of new mappings of row data take place?
The same applies every where below.

Reply:
Please review again and see if its a more suitable description.